tests/unit_tests/config/test_parser_error_collection.py

```diff
@@ -959,7 +959,6 @@ def test_that_executable_directory_errors(dirname):
 RSH_COMMAND qqqwe
 MAX_RUNNING_RSH assadsa 3 12w qwsa
 
-LSF_SERVER
```

Comment:
Should it be kept here? This looks like a file containg lots of ancient and now removed keywords..

Reply:
Yes, the test should be cleaned up to remove the keywords that are no longer in use and have been removed. I will create an issue for it.